plugins/inputs/snmp/CONFIG-EXAMPLES.md

```diff
@@ -63,3 +63,69 @@ Here is the configuration that you add to your `telegraf.conf`:
       oid = "IF-MIB::ifDescr"
       is_tag = true
 ```
+
+
+See below for SNMPv3 Examples of configs against Cisco Switches: 
+
+```
+[[inputs.snmp]]
+  agents = [AGENTS]
+  version = 3
+  auth_password = "TOKEN"
+  auth_protocol = "MD5"
+  priv_password = "TOKEN"
+  priv_protocol = "AES"
+  sec_name = "test"
+  sec_level = "authPriv"
+  max_repetitions = 10
+  timeout = "10s"
+  retries = 1
+  name_prefix = "dev."
+  
+  [[inputs.snmp.field]]
+    name = "hostname"
+    oid = ".1.3.6.1.2.1.1.5.0"
+    is_tag = true
+  [inputs.snmp.tagdrop]
+    ifName=["Vlan[0-9]*", "loopback*"]
+    fieldpass = ["ifInUcastPkts", "ifInNUcastPkts", "ifInDiscards", "ifInErrors", "ifOutUcastPkts", "ifOutNUcastPkts", "ifOutDiscards", "ifOutErrors", "ifType", "ifMtu", "ifSpeed", "ifAdminStatus", "ifOperStatus"]
+  [[inputs.snmp.table]]
+    name = "interfaceStats"
+    inherit_tags = ["hostname"]
+  [inputs.snmp.tagdrop]
```

Comment:
This needs to be combined with the `[inputs.snmp.tagdrop]` item above, a plugin can only have one tagdrop table.    I'm somewhat surprised this isn't a syntax error.